### DISCUSSION TITLE
How to disabel Power display in Webui for Shelly 2.5

### DISCUSSION THREAD

**Discussion by forumlw06** [arendst/Tasmota#18251] [Q&A]
Hello, i am pretty new with Tasmota (but already a great fan). 

I replaced my central shutter switch with a Tasmota 12.4 flashed Shelly 2.5 and everything works great.

The only thing i would like to change is to disable the display of the Power Settings in  Webui. In my case this is useless as there is no power consumption at all in my shelly, as the central switch does not move the shutters, but only triggers the up/down for the individual shutters and this then move the individual shutters up/down and consumes power. 

As the information is useless in webui i would like to disable it.

Shelly seems to use ADE7953 Power Sensor and this is "connected" (not sure if that is the right wording9 to GPIO 16.  If i configure the template and set GPIO 16 to "none", the power display in the main menu of Tasmota is gone. 

But as newbee i am unsure, if that really is the right way to disable the power display in main menu, or if there are better ways to achieve that. 
Does setting GPIO16 to "none" have any side effects on the "normal" operation as shutter controller?

Thanks for your help!

regards Rainer

**Comment by sfromis** [comment]
If what you want is not just removing energy monitoring data from the display, but completely (also in JSON payloads), no problem removing pin configs in the template. In that case, I'd suggest removing the I2C pins in addition to the IRQ pin. Of course, you could also have used a similar device without paying for the energy monitoring feature.

Anyway, Tasmota does not really provide much in tweaking for the web UI, as this is not meant as a place to visit during daily operation, with such being expected to be handled in a backend setup.

**Comment by forumlw06** [comment]
Thanks for your answer. With regards to the I2C pins you are talking about: do you mean to also set GPIO 12 (I2C SDA) and GPIO 14 (I2C SCL) to "none"? What are they used for? As mentioned i am new to that stuff and try to understand, what i am doing (probably something written in the docs, but with everything new, sometimes it is hard to understand/digest (especially without zero knowledge in microelectronics so far).

Can you imagine any side effects by setting the above 3 GPIOS to none? 

Kind regards
Rainer

**Reply by barbudor** [reply]
I2C is a on-board communication protocol/bus. It is used by Tasmota to communicate with the chip in charge of measuring the power.
If you remove those pins, as well, as the IRQ pin, Tasmota will just ignore there is a power management chip on the board and will just managed it as a relay system

No other side effect as on this device the I2C bus is used only for that purpose.

**Comment by forumlw06** [comment]
Thank you for the explanation and confirmation on the side effects!

This helps!

Regards Rainer
